authors/settings.py

```diff
@@ -1,33 +1,14 @@
-"""
-Django settings for authors project.
-
-Generated by 'django-admin startproject' using Django 1.11.14.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/1.11/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/1.11/ref/settings/
-"""
-
 import os
+from decouple import config, Csv
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
-
-# SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
 
```

Comment:
@NLSanyu I think also this secret_key value should also be exposed to the repository, can you also decouple it.

Reply:
@PatrickCmd May I have more clarification on what you mean by this?

Reply:
@PatrickCmd I have worked on it